temporal/api/cloud/cloudservice/v1/service.proto

```diff
@@ -179,6 +179,42 @@ service CloudService {
         };
     }
 
+    // Gets nexus endpoints
+    rpc GetNexusEndpoints(GetNexusEndpointsRequest) returns (GetNexusEndpointsResponse) {
+	      option (google.api.http) = {
+            get: "/cloud/nexus/endpoints",
+        };
+    }
+    
+    // Get a nexus endpoint
+    rpc GetNexusEndpoint(GetNexusEndpointRequest) returns (GetNexusEndpointResponse) {
+	      option (google.api.http) = {
+            get: "/cloud/nexus/endpoints/{endpoint_id}",
+        };
+    }
+
+    // Create a nexus endpoint
+    rpc CreateNexusEndpoint(CreateNexusEndpointRequest) returns (CreateNexusEndpointResponse) {
+        option (google.api.http) = {
+            post: "/cloud/nexus/endpoints",
+	          body: "*"
+        };
+    }
+
+    // Update a nexus endpoint
+    rpc UpdateNexusEndpoint(UpdateNexusEndpointRequest) returns (UpdateNexusEndpointResponse) {
+	      option (google.api.http) = {
+            post: "/cloud/nexus/endpoints/{endpoint_id}",
+	          body: "*"
+        };
+    }
+
+    // Delete a nexus endpoint
+    rpc DeleteNexusEndpoint(DeleteNexusEndpointRequest) returns (DeleteNexusEndpointResponse) {
+	      option (google.api.http) = {
+            delete: "/cloud/nexus/endpoints/{endpoint_id}",
+        };
+    }
```

Comment:
```suggestion
    }

```

Reply:
Is it time to add a formatter?

Reply:
I think that can be discussed on the primary `api` repo and can apply to this if/when it moves there